docs/source/conf.py

```diff
@@ -193,3 +194,9 @@
 plot_formats = [("png", 90)]
 plot_html_show_formats = False
 plot_html_show_source_link = False
+
+# sphinx plotly directive
+plotly_include_source = True
+plotly_formats = ["html"]
```

Comment:
[Question] I think this setting tells `plotly` to generate figures as HTML + JS, and I don't think it requires [`Orca`](https://github.com/plotly/orca) or [`Kaleido`](https://github.com/plotly/Kaleido). Is that correct?

Reply:
Yes, correct :)

Reply:
Thanks for your swift response!